build.xml

```diff
@@ -59,14 +381,72 @@
 		</exec>
 	</target>
 
-	<!-- to option copy file to webapps :) -->
+	<target name="clear-tomcat" description="Delete Tomcat workdir">
+		<delete dir="${env.CATALINA_HOME}/work/Catalina/localhost"/>
+		<delete file="${env.CATALINA_HOME}/webapps/ScadaBR.war"/>
+		<delete dir="${env.CATALINA_HOME}/webapps/ScadaBR"/>
+	</target>
+
+	<target name="build" description="Perform project build workflow" depends="
+		init-core,
+		init-static-dependencies,
+		init-ui-dependencies,
+		build-core,
+		build-ui
+	"/>
+
+	<target name="build-no-ui" description="Build just a core application" depends="
+		init-core,
+		init-static-dependencies,
+		build-core"/>
+
+	<target name="deploy" description="Deploy application to Tomcat workdir" depends="
+		clear-tomcat,
+		deploy-war,
+		deploy-tomcat"/>
+
+	<target name="run-build" description="Build and run application" depends="
+		build,
+		deploy,
+		run
+	"/>
+
+	<target name="run-build-no-ui" description="Build and run application without building frontend application" depends="
```

Comment:
build-deploy-run-no-ui

Reply:
Same as previous one

Reply:
build-run-no-ui ? 
It would be a good idea to follow the sequence of actions. Proposed pattern: {sequence actions} - {something distinctive}

Reply:
I would prefer organized tasks than the sequence of actions, because when I want to start application I would rather search for the `"run"` task and choose from available options instead of searching for that one which ends with "run". When you know the final goal what do you want to do it is much easier to find a specific task in that way.
![image](https://user-images.githubusercontent.com/22638815/116063227-07673e80-a685-11eb-88a3-ec48dc4db967.png)